website/docs/installation/configuration.mdx

```diff
@@ -71,16 +71,38 @@ To check if your config has been applied correctly, you can run the following co
 
 ## Redis Settings
 
--   `AUTHENTIK_REDIS__HOST`: Hostname of your Redis Server
--   `AUTHENTIK_REDIS__PORT`: Redis port, defaults to 6379
--   `AUTHENTIK_REDIS__PASSWORD`: Password for your Redis Server
--   `AUTHENTIK_REDIS__TLS`: Use TLS to connect to Redis, defaults to false
--   `AUTHENTIK_REDIS__TLS_REQS`: Redis TLS requirements, defaults to "none"
--   `AUTHENTIK_REDIS__DB`: Database, defaults to 0
--   `AUTHENTIK_REDIS__CACHE_TIMEOUT`: Timeout for cached data until it expires in seconds, defaults to 300
--   `AUTHENTIK_REDIS__CACHE_TIMEOUT_FLOWS`: Timeout for cached flow plans until they expire in seconds, defaults to 300
--   `AUTHENTIK_REDIS__CACHE_TIMEOUT_POLICIES`: Timeout for cached policies until they expire in seconds, defaults to 300
--   `AUTHENTIK_REDIS__CACHE_TIMEOUT_REPUTATION`: Timeout for cached reputation until they expire in seconds, defaults to 300
+-   `AUTHENTIK_REDIS__HOST`: Redis server host when not using configuration URL
+-   `AUTHENTIK_REDIS__PORT`: Redis server port when not using configuration URL
+-   `AUTHENTIK_REDIS__DB`: Redis server database when not using configuration URL
+-   `AUTHENTIK_REDIS__USERNAME`: Redis server username when not using configuration URL
+-   `AUTHENTIK_REDIS__PASSWORD`: Redis server password when not using configuration URL
+-   `AUTHENTIK_REDIS__TLS`: Redis server connection using TLS when not using configuration URL
+-   `AUTHENTIK_REDIS__TLS_REQS`: Redis server TLS connection requirements when not using configuration URL
+
+## Result Backend Settings
+
+-   `AUTHENTIK_RESULT_BACKEND__URL`: Result backend configuration URL, uses [the Redis Settings](#redis-settings) by default
+
+## Cache Settings
+
+-   `AUTHENTIK_CACHE__URL`: Cache configuration URL, uses [the Redis Settings](#redis-settings) by default
+-   `AUTHENTIK_CACHE__TIMEOUT`: Timeout for cached data until it expires in seconds, defaults to 300
+-   `AUTHENTIK_CACHE__TIMEOUT_FLOWS`: Timeout for cached flow plans until they expire in seconds, defaults to 300
+-   `AUTHENTIK_CACHE__TIMEOUT_POLICIES`: Timeout for cached policies until they expire in seconds, defaults to 300
+-   `AUTHENTIK_CACHE__TIMEOUT_REPUTATION`: Timeout for cached reputation until they expire in seconds, defaults to 300
+
+    :::info
+    `AUTHENTIK_CACHE__TIMEOUT_REPUTATION` only applies to the cache expiry, see [`AUTHENTIK_REPUTATION__EXPIRY`](#authentik_reputation__expiry) to control how long reputation is persisted for.
+    :::
+
+## Channel Settings
```

Comment:
We should probably explain for the docs what "Channel" means

```suggestion
## Websocket Settings
```

Reply:
I would like to keep the reference to Django Channel Layers as it makes it much easier to lookup should any questions arise.